static/solo-cta/js/mobile/modal/index.js

```diff
@@ -0,0 +1,296 @@
+        slickScript.src = config.baseUrl + '/solo-cta/js/vendor/slick-min.js'
+        slickScript.onload = onSlickLoad;
+        body.appendChild(slickScript);
+    };
+
+    var loadProducts = function(vid, cback) {
+        if (!vid || !cback) {
+            return;
+        }
+        
+        Utils.loadScript({
+            base: config.api_base_url + '/videos/' + vid + '/products',
+            params: {
+                'X-login-id': config.loginId,
+                o: config.products_order_by,
+                od: config.products_order_direction
+            }
+        }, cback);
+    };
+
+    var onNoProducts = function(){
+        Utils.addClass(mainEl,'tvp-no-products');
+        Utils.sendMessage({
+            event: eventPrefix + ':widget_modal_no_products'
+        });
+    };
+    var onProducts = function(){
+        Utils.removeClass(mainEl,'tvp-no-products');
+        Utils.sendMessage({
+            event: eventPrefix + ':widget_modal_products'
+        });
+    };
+
+    var onLoadProducts = function(data) {
+        data.length ? onProducts() : onNoProducts();
+        render(data);
+    };
+
+    var onPlayerNext = function(next) {
+        if (config.merchandise && next) {
+            loadProducts(next.assetId, onLoadProducts);
+        }else{
+            onNoProducts();
+        }
+
+        Utils.sendMessage({
+            event: eventPrefix + ':player_next',
+            next: next
+        });
+    };
+
+    var onPlayeResize = function() {
+        Utils.sendMessage({
+            event: eventPrefix + ':widget_modal_resize',
+            height: getWidgetHeight() + 'px'
+        });
+    };
+
+    var initializePlayer = function() {
+        var playerConfig = Utils.copy(config);
+        
+        playerConfig.data = config.channel.videos;
+        playerConfig.onResize = onPlayeResize;
+        playerConfig.onNext = onPlayerNext;
+
+        var player = new Player('tvp-player-el', playerConfig, config.clicked);
+
+        player.initialize();
+    };
+
+    //This should be done lazily.. after the important stuff
+    var initializeAnalytics = function(){
+        analytics =  new Analytics();
+        analytics.initConfig({
+            domain: location.hostname || '',
+            logUrl: config.api_base_url + '/__tvpa.gif',
+            loginId: config.loginId,
+            firstPartyCookies: config.firstpartycookies,
+            cookieDomain: config.cookiedomain
+        });
+    };
+
+    function initialize() {
+        initializePlayer();
+        initializeAnalytics();
+
+        if (config.merchandise) {
+            loadProducts(config.clicked, onLoadProducts);
+        }else{
+            onNoProducts();
+        }
+
+        Utils.sendMessage({
+            event: eventPrefix + ':widget_modal_initialized',
+            height: getWidgetHeight() + 'px'
+        });
+    };
+
+    var depsCheck = 0;
+    var deps = ['TVPage', '_tvpa', 'jQuery', 'Utils', 'Analytics', 'Player'];
```

Comment:
Please remove `_tvpa` no need to wait for it, _analytics.js_ singleton was updated to do lazy tracking